src/command_modules/azure-cli-batch/HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+3.0.0 (2017-05-29)
+++++++++++++++++++
+
+* Updated to Batch SDK 3.0.0 with support for low-priority VMs in pools.
```

Comment:
I know it is not exposed to command line. However, do we need to mention more update? for example, outfiles.